dispatch/tests/test_execute.py

```diff
@@ -0,0 +1,8 @@
+from dispatch.core.management import execute_from_command_line
+from django.test import TestCase
+
+class execute_from_command_lineTest(TestCase):
```

Comment:
In Python it's convention to camelcase all class names. We can probably shorten this as well:

```py
class CommandLineTest(TestCase):